src/common/options/rgw.yaml.in

```diff
@@ -2395,6 +2395,13 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_sync_data_full_inject_err_probability
```

Comment:
this has been useful for manual testing, but we'll want a test case that enables it explicitly. i don't want to delay merge though - can we follow up with another pr?

Reply:
sure, will follow up with a pr.